src/tests/navbar.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Navbar from '../components/Navbar';
+
+test('renders Navbar component correctly', () => {
+  const { container } = render(<Navbar />);
+  expect(container).toMatchSnapshot();
```

Comment:
- [ ]👋 Hello,

Great job on implementing unit tests for your React components! It's fantastic to see that you're taking testing seriously. However, I've noticed that some of your tests are failing. Let's take a closer look at the issues and work on fixing them together. 😊

![Screenshot from 2023-09-19 18-16-43](https://github.com/zhorabay/Math-magicians/assets/112261237/30d003eb-ee47-4039-b0ee-1bc49d7f4e00)